src/components/icon/icons/bulb.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Bulb({
+  stroke,
+  fill,
+  strokeWidth,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg style={style} {...rest} viewBox={'0 0 12 20'}>
```

Comment:
Set `viewBox` in defaultProps instead, see this for example: https://github.com/nordnet/nordnet-ui-kit/blob/master/src/components/icon/icons/checkmark.jsx#L34-L39